src/components/home/elements/TxInfoModal.vue

```diff
@@ -59,7 +59,7 @@
              :class="txClass + '-amount'">{{ txIcon === 'sent' ? '-' : txIcon === 'received' ? '+' : '' }}{{ formatter(txAmount) }} VSYS</div>
       </div>
       <div class="tx-address">
-        <label>{{ (txIcon === 'received' || txIcon === 'leased in' || txIcon === 'leased in canceled') ? 'From' : 'To' }}</label>
+        <label>{{ (txIcon === 'leased out' || txIcon === 'leased out canceled') ? 'To' : 'From' }}</label>
```

Comment:
`From` and `To` is wrong for Sending Tx record.
I think use icon to define `From` and `To` is a bad idea. It'd better use 2 rows to show `From` and `To`. If the no recipient, hide `To`.